html/book.hva

```diff
@@ -10,7 +11,8 @@
 \newstyle{.part}{margin:2ex auto;text-align:center}
 \@makesection
   {\chapter}{-1}{chapter}
-   {\@open{h1}{\@book@attr{chapter}}}{\chaptername~\thechapter}{\quad}{\@close{h1}}
+   {\@open{h1}{\@book@attr{chapter}}}%
+   {\ifthenelse{\equal{\chaptername}{}}{}{\chaptername~}\thechapter}{\quad}{\@close{h1}}
```

Comment:
There exists the primitive `\ife .... \else .... \fi`. Using it would prevent loading the ifthen package.
```suggestion
   {\ife\chaptername{}\else\chaptername~\fi\thechapter}{\quad}{\@close{h1}}
```